CKAN-meta/build.sh

```diff
@@ -52,7 +52,16 @@ create_dummy_ksp() {
     fi
 
     # Create dummy install.
-    mono ckan.exe ksp fake --set-default --headless "$KSP_NAME" dummy_ksp "$KSP_VERSION" 1.1.0
+    # The DLCs are simulated depending on the version.
+    if versions_less_or_equal "$KSP_VERSION" "1.4.0"
+    then
+        mono ckan.exe ksp fake --set-default --headless "$KSP_NAME" dummy_ksp "$KSP_VERSION"
+    elif versions_less_or_equal "$KSP_VERSION" "1.7.1"
```

Comment:
I think we need to flip the logic to get the right answer for the edge cases out of a less_or_equal operation. This would fail to generate BG for 1.7.1.

```sh
if versions_less_or_equal 1.7.1 "$KSP_VERSION"
then
    # MH + BG
elif versions_less_or_equal 1.4.0 "$KSP_VERSION"
then
    # Just MH
else
    # No DLC
fi
```

Reply:
You're right.
But shouldn't it work like this too?:
```
if versions_less_or_equal "$KSP_VERSION" 1.3.1
then
    # No DLC
elif versions_less_or_equal "$KSP_VERSION" 1.7.0
then
    # Just MH
else
    # MH + BG
fi
```

Reply:
Yes, either way would work. I think it's clearer to have the version when the DLC was released in the code rather than the previous version.

Reply:
Okay, makes sense.
I was unsure about when MH was released and read 1.4 in a lot of places, also that's what in our CKAN code is (-> needs fixing).
But now looking at that [Steam Post](https://steamcommunity.com/games/220200/announcements/detail/1678026870107657334) or at the [wiki](https://wiki.kerbalspaceprogram.com/wiki/1.4.1) convinces me that it was 1.4.1 indeed :smiley: